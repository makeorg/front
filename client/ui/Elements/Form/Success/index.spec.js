import React from 'react';
import renderer from 'react-test-renderer';
import { FormSuccessMessage } from './index';

jest.mock('Client/ui/Svg/elements', () => ({
  SvgCheck: 'SvgCheck',
}));

describe('FormSuccessMessage', () => {
  it('must match the snapshot', () => {
    const component = renderer.create(<FormSuccessMessage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
