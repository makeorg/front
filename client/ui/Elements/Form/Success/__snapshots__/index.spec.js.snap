// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSuccessMessage must match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  margin-top: 10px;
  font-size: 12px;
  line-height: 18px;
}

<p
  className="c0"
  tabIndex={0}
>
  <SvgCheck
    style={
      Object {
        "display": "inline-flex",
        "fill": "rgba(80,122,31,1)",
        "fontSize": "16px",
        "marginRight": "5px",
      }
    }
  />
  common.form.messages.submit_success
</p>
`;
